nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/message/BulkCommand.java

```diff
@@ -181,7 +204,10 @@ public Builder repository(String name) {
 
         /**
          * User running the bulk action
+         *
+         * @deprecated 11.1 use constructor with username instead
```

Comment:
`@deprecated 11.1 use constructor with username instead` -> `@deprecated since 11.1`, use {@link #Builder(String, String, String)} constructor with username instead.